src/ucs/sys/event_set.h

```diff
@@ -15,12 +15,18 @@
  */
 typedef struct ucs_sys_event_set ucs_sys_event_set_t;
 
+/**
+ * ucs_event_set_extra_arg used in ucs_event_set_wait callback function.
+ *
+ */
+typedef void* ucs_event_set_extra_arg;
 
 /**
  * ucs_event_set_handler call this handler for notifying event
  *
  */
-typedef void (*event_set_handler_t)(int fd, int event);
+typedef void (*event_set_handler_t)(int fd, int event,
```

Comment:
btw, can rename to `ucs_event_set_handler_t` ? we should keep `ucs_` prefix for all types